NitroxModel/DataStructures/GameLogic/PDASaveData.cs

```diff
@@ -0,0 +1,194 @@
+﻿using System;
+using UnityEngine;
+using System.Collections.Generic;
+using ProtoBuf;
+
+namespace NitroxModel.DataStructures.GameLogic
+{
+    [Serializable]
+    [ProtoContract]
+    public class PDASaveData
+    {
+
+        [ProtoMember(1)]
+        public PDADataPartial PDADataPartial { get; set; }
+        [ProtoMember(2)]
+        public PDADataComplete PDADataComplete { get; set; }
+        [ProtoMember(3)]
+        public PDADataEnciclopedia PDADataEnciclopedia { get; set; }
+        [ProtoMember(4)]
+        public PDADataknownTech PDADataknownTech { get; set; }
+
+        public PDASaveData()
+        {
+
+        }
+    }
+
+    [Serializable]
+    [ProtoContract]
+    public class PDADataPartial
+    {
+        public PDADataPartial()
+        {
+            // For serialization purposes
+        }
+
+        [ProtoIgnore]
+        private List<PDA_Entry> pdaEntryPartial = new List<PDA_Entry>();
+
+        [ProtoMember(1)]
+        public List<PDA_Entry> Serializable
```

Comment:
Serializable fields should only be used for complex datatypes that protobuf doesn't know how to handle (example: a Dictionary<SomeObject, AnotherObject>).  Because protobuf handles lists correctly, you can simply set the above field (pdaEntryPartial) to public and ProtoMember(1).  

Please do this for other classes throughout this review as well.